.github/workflows/test.yml

```
@@ -18,5 +18,5 @@ jobs:
     - uses: ./
       with: 
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        persons: "sinchang"
+        persons: "rowi1de"
```

To keep the possibility to test in own PRs
```suggestion
        persons: "rowi1de, sinchang"
```

---

oops... error `Review cannot be requested from pull request author.`